views/routes/routes.js

```diff
@@ -14,12 +14,13 @@ export const routes = {
   admin: `${basePath}/admin`,
   logout: `${basePath}/logout`,
   studyDetails: `${basePath}/study-details`,
-  chart: (chart_id) => `${basePath}/charts/${chart_id}`
+  chart: (chart_id) => `${basePath}/charts/${chart_id}`,
+  editChart: (chart_id) => `${basePath}/charts/edit/${chart_id}`,
```

Comment:
This is the route name convention I've seen in the past. We can go either way, just figured I'd call it out.

```suggestion
  editChart: (chart_id) => `${basePath}/charts/${chart_id}/edit`,
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/118/commits/a02b5f9495eed90745bb37c6f90df4df8e963870